### PR TITLE
Add nuclear parser to JP-SK production parser 

### PR DESCRIPTION
**Pull request by wobniarin**
## Issue
the parser for JP-SK only accounts for unknown and solar, and nuclear has to be fetched from a different website (nuclear power plant operator) and reading the information from a GIF. 

## Description
This PR creates a specific parser for JP-SK where nuclear power can be fetched and added to the data point for a given timestamp.

This is the [website](https://www.yonden.co.jp/energy/atom/ikata/ikt722.html) we want to extract nuclear power production from 

image

We also include tests to validate that we can parse the nuclear power production

### Preview

Result after implementing the parser 
image

### Double check

- [X] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [X] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### Expected reviews 

@VIKTORVAV99  and @unitrium I added you two since I think you can provide valuable feedback on this parser and the tests I have implemented.

### CODE REVIEW

**Review by VIKTORVAV99 — COMMENTED** [review]
Just some small suggestions but otherwise it looks good!

(We don't particularly like pytz either 😅)

**Comment by VIKTORVAV99** [comment]
Just some notes that are not part of the review. I know this PR is not using the ParserEvent classes which I think is okay since the core JP parser don't either. When this is updated this parser should be updated in the same PR.

**Comment by wobniarin** [comment]
> Just some notes that are not part of the review. I know this PR is not using the ParserEvent classes which I think is okay since the core JP parser don't either. When that is updated this parser should be updated in the same PR.

Makes sense, thanks for the heads-up! :)

File: parsers/JP_SK.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    filtered_img_tags = [tag for tag in images_links if "ikt721-1" in tag["src"]]
+    if len(filtered_img_tags) == 0:
+        raise Exception("No image found")
+    img_url = IMAGE_CORE_URL + filtered_img_tags[0]["src"]
+    return img_url
+
+
+def get_nuclear_power_value_and_timestamp_from_image_url(
+    img_url: str, session: Session
+) -> (float, datetime):
```
Comment (VIKTORVAV99) [inline parsers/JP_SK.py:80]:
```suggestion
def get_nuclear_power_value_and_timestamp_from_image_url(
    img_url: str, session: Session
) -> tuple[float, datetime]:
```

File: parsers/JP_SK.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    """This method reads the image from the image url and extracts nuclear power and timestamp from it."""
+    response_image = session.get(img_url)
+    image = Image.open(BytesIO(response_image.content))
+    width, height = image.size
+    img = image.crop((0, 0, width, height))
+    text = image_to_string(img)
```
Comment (VIKTORVAV99) [inline parsers/JP_SK.py:86]:
```suggestion
    text = str(image_to_string(img))
```

**Review by VIKTORVAV99 — APPROVED** [review]
Just two tiny comments to fix two type errors, they don't change any functionality though so approving.

Nice work!

**Pull request merged**